contracts/modules/common/LimitUtils.sol

```diff
@@ -56,14 +56,15 @@ library LimitUtils {
         internal
     {
         ILimitStorage.Limit memory limit = _lStorage.getLimit(_wallet);
-        // solium-disable-next-line security/no-block-members
         uint256 currentLimit = currentLimit(limit);
-        ILimitStorage.Limit memory newLimit = ILimitStorage.Limit(
-            safe128(currentLimit),
-            safe128(_targetLimit),
+        ILimitStorage.Limit memory newLimit;
+        if (_targetLimit <= currentLimit) {
+            // solium-disable-next-line security/no-block-members
+            newLimit = ILimitStorage.Limit(safe128(_targetLimit), uint128(0), safe64(now));
```

Comment:
It is not better to set the `Limit.changeAfter` property to `0` instead of `now`? Both work logic wise but it seemed more natural when the `Limit.pending` is `0` for `Limit.changeAfter` to be `0` as well.
Same as this logic https://github.com/argentlabs/argent-contracts/pull/138/files#diff-252645bba53001a15e101c0e5845da2eR127

Reply:
Agreed. I hesitated between both options but the one you suggest is more consistent.

Reply:
So actually the logic in https://github.com/argentlabs/argent-contracts/pull/138/files#diff-252645bba53001a15e101c0e5845da2eR127 is not consistent. I will update it so that when the limit is changed immediately the `changeAfter` parameter is always set to `now`.

Reply:
Note that if we set `limit.changeAfter` to `now` then we must also set `limit.pending` to `_targetLimit` if we want `LimitUtils.currentLimit` to return the correct value.

Reply:
> Note that if we set limit.changeAfter to now then we must also set limit.pending to _targetLimit if we want LimitUtils.currentLimit to return the correct value.

I don't think that's the case: the `if` in `currentLimit()` will return `false` since `now < now` is `false`. So it seems we could set `limit.pending` to `0`